generators/entity-server/templates/src/main/java/package/common/get_all_template.ejs

```diff
@@ -26,7 +26,9 @@
 _%>
 <%_ if (jpaMetamodelFiltering) { _%>
     public ResponseEntity<List<<%= instanceType %>>> getAll<%= entityClassPlural %>(<%= entityClass %>Criteria criteria<% if (!paginationNo) { %>, @org.springdoc.api.annotations.ParameterObject Pageable pageable<% if (reactive) { %>, ServerHttpRequest request<% } %><% } %>) {
+      if (log.isDebugEnabled()) {  
         log.debug("REST request to get <%= entityClassPlural %> by criteria: {}", criteria.toString().replaceAll("[\n\r\t]", "_"));
+      }
```

Comment:
```suggestion
        log.debug("REST request to get <%= entityClassPlural %> by criteria: {}", () => criteria.toString().replaceAll("[\n\r\t]", "_"));
```
I find the supplier solution more elegant 😄. What do you think

Reply:
yes 👍